bin/hammer-complete

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+require 'English'
+@project_root = File.expand_path('../..', __FILE__)
+$LOAD_PATH.unshift(File.join(@project_root, 'lib'))
+system('which hammer > /dev/null 2>&1')
+found = $CHILD_STATUS.success?
+HAMMER = found ? 'hammer' : File.join(project_root, 'bin/hammer')
+
+require 'yaml'
+
+def cache_file
+  cache_file = nil
+  [
+    File.join(@project_root, 'config/cli_config.template.yml'),
+    '~/.hammer/cli_config.yml',
+    '/etc/hammer/cli_config.yml',
+    "#{::RbConfig::CONFIG['sysconfdir']}/hammer/cli_config.yml"
+  ].uniq.map { |p| File.expand_path(p) }.select { |f| File.exist?(f) }.reverse
+   .each do |file|
+    File.open(file) do |f|
+      config = YAML.load(f)
+      cache_file = config[:completion_cache_file] || cache_file
+    end
+  end
+  cache_file
+end
+
+completion_cache_file = cache_file || '~/.cache/hammer_completion.yml'
```

Comment:
@ofedoren it may spare some cycles if we make it more opinionated and define it as `ENV['HAMMER_COMPLETION_CACHE'] || '~/.cache/hammer_completion.yml'`. No idea how much time we spend with the config loading and parsing. We could do similarly for the hammer command name (`HAMMER`).